tests/installation/partitioning_raid.pm

```diff
@@ -70,6 +70,7 @@ sub addraid {
 
     # add
     send_key $cmd{add};
+    save_screenshot; # for debug purpose
```

Comment:
You need to call `tools/tidy`. Wouldn't it be better to move that after `wait_still_screen` to ensure the "still screen" is catched?